packages/database/package.json

```diff
@@ -13,7 +13,7 @@
     "init": "supabase login",
     "dev": "supabase start && tsx scripts/createEnv.ts && supabase functions serve",
     "stop": "supabase stop",
-    "check-types": "npm run lint && supabase stop && npm run dbdiff",
```

Comment:
@maparent not sure if you were using this before, if so, let's move it to another command and keep `check-types` consistent

Reply:
It is found in two other commands, it's fine to remove.